### DISCUSSION TITLE
Create a demo mod with each major release

### DISCUSSION THREAD

**Discussion by Traderain** [WolvenKit/WolvenKit#958] [General]
So this is open for discussion but I would like to start doing a thing where we "demo" each bigger release with an example mod. I volunteer to make this mod but I would like to change it so we are a bit more user usability driven with releases.
So we can start this with 8.7 the "REDmod support" release. So here the following would be:
- [x] We create a mod
   - [ ] Replacing an animation
   - [x] REDmod script change
   - [x] REDmod tweak change
   - [x] REDmod audio event override
- [x] We update the documentation
- [x] We do a WolvenKit release stream with the mod on twitch where we show-off that hey this is WolvenKit 8.7 this is what you can do with it.
- [x] We make a release video about the mod and maybe some of the bigger features we make
@ja-to @Moonded @rfuzzo @jackhumbert @seberoth 

![image](https://user-images.githubusercontent.com/6411732/195056156-99f4a809-4312-4e44-8bd2-ebdfc0bbc6a8.png)

Opinnions? Other ideas?

**Comment by seberoth** [comment]
Could also do some "simple" Q&A part in the stream. Like everyone of us in a discord call which is also on stream and we answer questions from the chat

**Comment by Traderain** [comment]
Sooo this went more or less good. So good job everyone!
